generator/metricbeat/{beat}/README.md

```diff
@@ -35,11 +35,11 @@ This updates all fields and docs with the most recent changes.
 
 ## Use vendoring
 
-We recommend to use vendoring for your beat. This means the dependencies are put into your beat folder. The beats team currently uses [glide](https://github.com/Masterminds/glide) for vendoring.
+We recommend to use vendoring for your beat. This means the dependencies are put into your beat folder. The beats team currently uses [govendor](https://github.com/kardianos/govendor) for vendoring.
 
 ```
-glide init
-glide update --quick
+govendor init
+govendor update
 ```
```

Comment:
Currently `clean-vendor` only exists only on the `beats` level and not as part of each beat. It definitively brings up the question if we should introduce it there. TBH `govendor` is on the of the reasons I'm even thinking we could probably remove `clean-vendor` in general as it already removes lots of files like the test files. So this becomes obsolete.